README.md

```diff
@@ -120,8 +125,9 @@ Admin endpoints are:
 `/__health`
 
 There are following checks are performed when the `/__health` is called:
-1. **content-collection-neo4j-rw** connectivity check
-2. **document-store-api** connectivity check
-3. **kafka** connectivity check and topic existence check
+1. **relations-api** connectivity check
+2. **content-collection-neo4j-rw** connectivity check
+3. **document-store-api** connectivity check
+4. **kafka** connectivity check and topic existence check
```

Comment:
The existence of the kafka topic is no longer checked.

Reply:
true